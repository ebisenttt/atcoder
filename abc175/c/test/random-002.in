-674577672706889 812786407297667 663661335502490
