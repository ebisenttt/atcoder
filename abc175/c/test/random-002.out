10916337204399
